src/libraries/System.ComponentModel.Composition/src/System.ComponentModel.Composition.csproj

```diff
@@ -10,7 +10,7 @@
     <NoWarn Condition="'$(TargetFramework)' == 'netcoreapp2.0'">$(NoWarn);nullable</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="TypeForwards.cs" />
+    <Compile Condition="'$(TargetFramework)' != 'netstandard2.0'" Include="TypeForwards.cs" />
```

Comment:
(nit) Can this `Compile` line just move into the next `ItemGroup`? It has the same condition.